### ISSUE TITLE
Add `lighthousebot` to site deployments

### ISSUE THREAD

**Issue by adamzerella** [govau/design-system-site#487]
We currently run a suite of accessibility tests with a tool called `pa11y`. Lighthouse is a really robust tool that looks quite promising. It's currently only supported with `TravisCI` deployments, we'd need to switch from `CircleCI`.

See: https://github.com/GoogleChromeLabs/lighthousebot

**Comment by sukhrajghuman** [comment]
Possible option without switching to travis:

Have checked out the [npm lighthouse package](https://www.npmjs.com/package/lighthouse).

It can generate a JSON report which gives a score (from 0 to 1) for each of the categories. It's pretty hard to get 1 for all of the categories. But we could set a benchmark for each category. E.g. `Interactive > 0.7` or `Accessibility > 0.95`, and if we don't meet this standard the build will fail. Could be a good regression tool for any future changes.

Pretty heavy package though. Will slow things down a fair bit

**Comment by sukhrajghuman** [comment]
Won't be adding this to our deployments anytime soon.
